pkg/errors/typed.go

```diff
@@ -127,7 +127,7 @@ func NewCorruptedConfigError(format, contextName, configFile string) CLITypedErr
 	return &CorruptedConfigError{
 		errorMsg: fmt.Sprintf("corrupted CLI config: %v", errorWithStackTrace),
 		suggestionsMsg: fmt.Sprintf("Your configuration file \"%s\" is corrupted.\n"+
-			"Remove config file, and run `confluent login` or `confluent context create`.\n"+
+			"Remove it, and run `confluent login`.\n"+
```

Comment:
Nit: More precise wording
```suggestion
			"Delete it, and run `confluent login`.\n"+
```